src/main/drivers/accgyro_fake.c

```diff
@@ -20,6 +20,28 @@
 
 #include "platform.h"
 
+#if defined(SIMULATOR_BUILD) && defined(SIMULATOR_MULTITHREAD)
+#include <stdio.h>
+#include <pthread.h>
+#define printf printf
+#define sprintf sprintf
+
+#define ACC_LOCK pthread_mutex_unlock(&accUpdateLock)
```

Comment:
It may be better to use more function-like syntax:
```
#define ACC_LOCK pthread_mutex_unlock(&accUpdateLock)
and then
ACC_LOCK();
```
Or use normal static functions, they will be inlined